_data/ultimaterex/ultimaterex.md

```diff
@@ -1,5 +1,5 @@
 name: Rex Serubii
```

Comment:
Please add --- as shown in the template as the preview needs to be in the form of a table

Reply:
I missed that somewhere down the line, thanks!